lib/shopify_transporter/exporters/magento/customer_exporter.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module ShopifyTransporter
+  module Exporters
+    module Magento
+      class CustomerExporter
+        attr_accessor :client, :store_id
```

Comment:
Do we want the class user to be able to modify these after they're set by the initializer?  If not, maybe change to `attr_reader`.

Reply:
Or maybe we don't even need `attr_` anything.